mesa-quilt

```diff
@@ -18,17 +18,73 @@
 # 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
 
 d=. ; while [ ! -d "$d/nx-X11/extras" ] && [ "$(readlink -e "$d")" != "/" ]; do d="$d/.."; done
```

Comment:
> I don't see why climbing up until / gets hit makes any sense...

@uli42: I have a copy of the mesa-quilt script in ~/bin/. So I can use it from anywhere in the nx-libs tree and don't need to give the full path to the script either. Very handy.